doc/source/methods/PartialDependence.ipynb

```diff
@@ -0,0 +1,265 @@
+{
```

Comment:
<p>First sentence, add ":" to the end?</p><p>"meta data" -&gt; "metadata"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>